### ISSUE TITLE
Who determines what attestation APIs are acceptable?

### ISSUE THREAD

**Issue by rektide** [RupertBenWiser/Web-Environment-Integrity#14]
This proposal is about creating a way to create attestation tokens. Currently there are at least [two pieces of information proposed](https://github.com/RupertBenWiser/Web-Environment-Integrity/blob/main/explainer.md#what-information-is-in-the-signed-attestation), signed by the attester: a verdict on trustability, and an attester identity.

How do new browsers ever get trusted? If we rely on each site operator to determine which attesters to trust, what hope is there for other browsers to ever get off the ground? It seems unlikely that smaller browsers will gain much recognition here. This seems to favor only extremely large entrenched forces.

**Comment by jfmcbrayer** [comment]
I think the question of "what browsers are acceptable" is kind of to the side of "what attesters are acceptable". It's expected that there will be a fairly small number of attesters - realistically, Google, Microsoft, and Apple. Given that attesters are a practically closed set, doesn't this boil down to a proposal to let those entities decide who is "allowed" on the web?

**Comment by pinobatch** [comment]
Would distributors of GNU/Linux and AOSP/Linux systems be eligible attesters?
- Canonical (Ubuntu)
- Red Hat (RHEL, CentOS Stream, Fedora)
- SUSE
- Aaron Griffin (Arch Linux)
- Debian Project
- System76 (Pop! OS)
- Purism (Librem)
- Pine64 (Pinebook, PinePhone)
- Amazon (Fire tablet)

**Comment by AshtonKem** [comment]
> I think the question of "what browsers are acceptable" is kind of to the side of "what attesters are acceptable". It's expected that there will be a fairly small number of attesters - realistically, Google, Microsoft, and Apple, plus maybe some providers of corporate "endpoint protection" spyware. Given that attesters are a practically closed set, doesn't this boil down to a proposal to let those entities decide who is "allowed" on the web?

Indeed. The entire "we don't let websites discriminate against browsers" seems like a rather paper thin fig leaf without specifying who the attestors are, and how new organizations can become one. 

Attestor vs browser is a distinction without a difference if the attestors are all major corporations with aligned economic interests.
